client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -843,9 +849,26 @@ export const ApplicationsTable: React.FC = () => {
                 {getColumnVisibility("tags") && (
                   <Th {...getThProps({ columnKey: "tags" })} width={10} />
                 )}
-                {getColumnVisibility("effort") && (
-                  <Th {...getThProps({ columnKey: "effort" })} width={10} />
-                )}
+                <Th {...getThProps({ columnKey: "effort" })}>
+                  <Flex
+                    flexWrap={{ default: "nowrap" }}
+                    spaceItems={{ default: "spaceItemsSm" }}
```

Comment:
This will ensure the icon is correctly aligned vertically.
<img width="439" alt="Screenshot 2024-04-24 at 12 03 57 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/b5b824e7-f3e0-411c-b195-5cf0f885c9ca">